google-cloud-firestore/src/main/java/com/google/cloud/firestore/CollectionGroup.java

```diff
@@ -0,0 +1,70 @@
+package com.google.cloud.firestore;
```

Comment:
Missing Copyright Header

Reply:
These appear to be why most of your builds are failing

Reply:
Oh, wow. Thank you. I ran the code formatter and it didn't produce any changes, so I was pretty confused here.

Reply:
Added